libs/hdf-converters/src/msft-secure-score-mapper.ts

```diff
@@ -0,0 +1,358 @@
+                    return result.length > 0;
+                  })()
+                ),
+                ...conditionallyProvideAttribute(
+                  'tiers',
+                  this.getProfiles(data.controlName || '').map(
+                    (profile) => profile.tier
+                  ),
+                  (() => {
+                    const result = this.getProfiles(data.controlName || '')
+                      .map((profile) => profile.tier)
+                      .filter((v) => Boolean(v));
+                    return result.length > 0;
+                  })()
+                ),
+                ...conditionallyProvideAttribute(
+                  'threats',
+                  _.uniq(
+                    this.getProfiles(data.controlName || '').map(
+                      (profile) => profile.threats
+                    )
+                  ),
+                  (() => {
+                    const result = this.getProfiles(data.controlName || '')
+                      .map((profile) => profile.threats)
+                      .filter((v) => Boolean(v));
+                    return result.length > 0;
+                  })()
+                ),
+                ...conditionallyProvideAttribute(
+                  'services',
+                  _.uniq(
+                    this.getProfiles(data.controlName || '').map(
+                      (profile) => profile.service
+                    )
+                  ),
+                  (() => {
+                    const result = this.getProfiles(data.controlName || '')
+                      .map((profile) => profile.service)
+                      .filter((v) => Boolean(v));
+                    return result.length > 0;
+                  })()
+                ),
+                ...conditionallyProvideAttribute(
+                  'rank',
+                  _.uniq(
+                    this.getProfiles(data.controlName || '').map(
+                      (profile) => profile.rank
+                    )
+                  ),
+                  (() => {
+                    const result = this.getProfiles(data.controlName || '')
+                      .map((profile) => profile.rank)
+                      .filter((v) => Boolean(v));
+                    return result.length > 0;
+                  })()
+                ),
+                ...conditionallyProvideAttribute(
+                  'userImpacts',
+                  _.uniq(
+                    this.getProfiles(data.controlName || '').map(
+                      (profile) => profile.userImpact
+                    )
+                  ),
+                  (() => {
+                    const result = this.getProfiles(data.controlName || '')
+                      .map((profile) => profile.userImpact)
+                      .filter((v) => Boolean(v));
+                    return result.length > 0;
+                  })()
+                )
+              }),
+              nist: ['SA-11', 'RA-5']
```

Comment:
instead of hardcoding this in, please use the constants defined here: https://github.com/mitre/heimdall2/blob/master/libs/hdf-converters/src/utils/global.ts#L12